addons/mail/static/src/core/common/thread.js

```diff
@@ -558,4 +558,10 @@ export class Thread extends Component {
             this.messageHighlight.scrollTo(el).then(() => (this.scrollingToHighlight = false));
         }
     }
+
+    get orderedMessages() {
+        return this.props.order === "asc"
+            ? [...this.props.thread.nonEmptyMessages]
```

Comment:
any reason to make a copy?

Reply:
patch adds the `from_message_id` to the list, we don't want to alter thread message list.

Reply:
It's a little strange to always have the parent message at the top, even when there is a gap in message list.
But it's not such a big deal, as it's pretty much not visible if there are other messages, and then there is load more.

Reply:
Agreed, not a big deal. It might be present during the load more but it will pretty much be invisible in this case.